pyseir/cli.py

```diff
@@ -265,8 +307,9 @@ def _build_all_for_states(
         state_fitter_map, fips=fips, states=states
     )
 
-    with Pool(maxtasksperchild=1) as p:
+    with Pool() as p:
```

Comment:
We use maxtasksperchild=1 in a bunch of places. Add a comment about why not here.